demo/script.js

```diff
@@ -0,0 +1,217 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
In the demo, the only minor comment I have is that the curve is drawn flipped vertically. It doesn't really matter much, but just to be consistent with the README and plotpath, and it it's not a huge hassle to change, maybe it would be worth making the curve start in the lower left corner.

Reply:
Maybe this is worth a separate issue. I'll open it and work on it.